x-pack/elastic-agent/CHANGELOG.asciidoc

```diff
@@ -36,6 +36,7 @@
 - Fix jq: command not found {pull}18408[18408]
 - Avoid Chown on windows {pull}18512[18512]
 - Remove fleet admin from setup script {pull}18611[18611]
+- Correctly report platform and family. {issue}18665[18665]
```

Comment:
incorrect pull request number 665 instead of diabolic 666

Reply:
I've linked the original issue https://github.com/elastic/beats/issues/18665 instead of this PR, and used `issue` instead of `pull` :)

:see_no_evil:

Reply:
oh cool 👍